rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java

```diff
@@ -75,6 +76,9 @@ public void handleRerun(String clusterName, String entityType, String entityName
                         "All retry attempt failed out of configured: "
                                 + attempts + " attempt for entity instance::");
             }
+        } catch (EntityNotRegisteredException ee) {
```

Comment:
In RetryHandler Failure Case also this exception might occur, later after 0.9 we should fix this cleanly, instead of handling in each and every listener. Even though i am not sure what's the best way to handle this cleanly.